mu4/scenes/inspector/internal/services/elementrepositoryservice.cpp

```diff
@@ -323,3 +329,20 @@ QList<Ms::Element*> ElementRepositoryService::findTexts() const
 
     return resultList;
 }
+
+QList<Ms::Element*> ElementRepositoryService::findTremolos() const
+{
+    QList<Ms::Element*> resultList;
+
+    for (Ms::Element* element : m_elementList) {
+        Ms::Tremolo* tremolo = Ms::toTremolo(element);
```

Comment:
You are casting to a tremolo without first checking the element type. So, of course, `CONVERT(Tremolo)` will trigger an assertion failure every time there are any elements selected which are not tremolos.

Reply:
Oh that was foolish of me, sorry!